pyfr/integrators/dual/multip.py

```diff
@@ -186,29 +196,34 @@ def _add(self, *args):
 
     @property
     def _mg_regidx(self):
-        return self._regidx[-3:]
+        return self._regidx[-2:] if self.level != self._order else [None]*2
```

Comment:
if self.level == self._order:
    raise AttributeError('_mg_regidx not defined when self.level == self._order')

return ...